tox.ini

```diff
@@ -1,26 +1,19 @@
 [tox]
 envlist =
-    py38,py37,py36,py35,py27,pypy,
-    {py27,py37}-flakes,
-    {py27,py37}-with_numpy,
-    {py27,py37}-with_ipython
-    {py27,py37}-with_pandas
+    py39,py38,py37,py36,py35,py27,pypy,
+    py37-flakes,
+    py37-coverage,
+    py37-with_numpy,
+    py37-with_ipython,
+    py37-with_pandas,
+    py37-with_jsonsschema,
+    py37-with_gmpy,
+    py37-with_all
```

Comment:
At some point we should presumably migrate py37-x to py38-x or py39-x, whenever the number of 3.8 or 3.9 users dominates the number of 3.7 users. I.e., those tests only need to run on one Python version, and it may as well be whatever is most popular...

Reply:
Yes, that's right. I don't know a way to express "current default python" (which would be set to py37) in the tox file. I'll add a comment.